po/es.po

```diff
@@ -30,29 +30,17 @@ msgstr ""
 "Plural-Forms: nplurals=2; plural=(n != 1)\n"
 "X-Generator: Gtranslator 3.34.0\n"
 
-msgid " To stop these messages, say STOP."
-msgstr ""
+msgid "To stop these messages, say STOP."
```

Comment:
Don't change the msgid, it will break the translation because the string is looked up with the space. It needs a space at the beginning anyway to separate from the rest of the message.